lib/acts_as_tenant.rb

```diff
@@ -109,6 +109,14 @@ def self.without_tenant(&block)
     self.current_tenant = old_tenant
     self.unscoped = old_unscoped
   end
+
+  def self.should_require_tenant?
+    if configuration.require_tenant.respond_to?(:call)
+      configuration.require_tenant.call
```

Comment:
Should we also make this `!!configuration.require_tenant.call` in case the lambda/proc returns something other than a boolean?